modules/ROOT/pages/create-outbound-message-flow.adoc

```diff
@@ -96,7 +100,7 @@ To configure acknowledgement settings for the outbound messages that use the mes
 . Do either of the following:
 * Select an existing endpoint at which to receive your partner's acknowledgement transaction.
 * Click *New* to create a new endpoint, as described in xref:create-endpoint.adoc[Creating Endpoints].
-. In the *Mark overdue after* field, accept the default value of *24 Hours* or configure when you want the outbound transaction to be marked as overdue for an acknowledgement, according to the SLAs you have with your trading partner.
+. In the *Mark overdue after* field, accept the default value of *24 Hours* or configure when you want the outbound transaction to be marked as overdue for an acknowledgement, according to the SLAs you have with your partner.
```

Comment:
...configure when to mark outbound transaction as overdue for an acknowledgement, according to the SLAs you have with your partner.

Reply:
OK